docs/tutorials/timeslices.md

```diff
@@ -0,0 +1,258 @@
+  --slice-duration 10000
+```
+
+This creates 6 time slices (0-10s, 10-20s, 20-30s, 30-40s, 40-50s, 50-60s), each with independent metrics.
+
+## Output Files
+
+When time slicing is enabled, AIPerf generates additional output files:
+
+### CSV Timeslice Export
+
+**File**: `artifacts/profile_export_aiperf_timeslices.csv`
+
+The CSV uses a "tidy" (long) format optimized for data analysis:
+
+```csv
+Timeslice,Metric,Unit,Stat,Value
+0,Time to First Token,ms,avg,45.23
+0,Time to First Token,ms,min,32.10
+0,Time to First Token,ms,max,78.45
+0,Time to First Token,ms,p50,44.20
+0,Time to First Token,ms,p90,65.30
+0,Time to First Token,ms,p95,70.15
+0,Time to First Token,ms,p99,76.80
+0,Inter Token Latency,ms,avg,12.34
+0,Inter Token Latency,ms,min,8.50
+...
+1,Time to First Token,ms,avg,46.78
+1,Time to First Token,ms,min,33.20
+...
+```
+
+**Format Details**:
+- **Timeslice**: Zero-indexed slice number (0, 1, 2, ...)
+- **Metric**: Human-readable metric name (e.g., "Time to First Token")
+- **Unit**: Measurement unit (ms, tokens/sec, etc.)
+- **Stat**: Statistical measure (avg, min, max, p50, p90, p95, p99)
+- **Value**: Numeric value formatted to 2 decimal places
+
+### JSON Timeslice Export
+
+**File**: `artifacts/profile_export_aiperf_timeslices.json`
+
+The JSON provides a hierarchical structure with all timeslices in a single file:
+
+```json
+{
+  "timeslices": [
+    {
+      "timeslice_index": 0,
+      "time_to_first_token": {
+        "unit": "ms",
+        "avg": 45.23,
+        "min": 32.10,
+        "max": 78.45,
+        "p50": 44.20,
+        "p90": 65.30,
+        "p95": 70.15,
+        "p99": 76.80
+      },
+      "inter_token_latency": {
+        "unit": "ms",
+        "avg": 12.34,
+        "min": 8.50,
+        "max": 18.90,
+        "p50": 12.10,
+        "p90": 15.80,
+        "p95": 16.50,
+        "p99": 17.90
+      },
+      ...
+    },
+    {
+      "timeslice_index": 1,
+      "time_to_first_token": {
+        "unit": "ms",
+        "avg": 46.78,
+        ...
+      },
+      ...
+    }
+  ],
+  "input_config": {
+    "model": "Qwen/Qwen3-0.6B",
+    "endpoint": "/v1/chat/completions",
+    ...
+  },
+  "total_slices": 6
+}
+```
+
+**Key Fields**:
+- `timeslices`: Array of slice objects, ordered by time
+- `timeslice_index`: Zero-indexed slice identifier
+- Each metric contains `unit` and available statistics
+- `input_config`: Benchmark configuration for reproducibility
+
+## Use Cases
+
+### Detecting Warm-up Effects
+
+Identify initial cold-start latency vs. steady-state performance:
+
+```bash
+aiperf profile \
+    --model Qwen/Qwen3-0.6B \
+    --endpoint-type chat \
+    --endpoint /v1/chat/completions \
+    --streaming \
+    --url localhost:8000 \
+    --benchmark-duration 120.0 \
+    --slice-duration 20.0 \
+    --concurrency 10 \
+    --synthetic-input-tokens-mean 200 \
+    --output-tokens-mean 100
+```
+
+Expected pattern: Higher latency in slice 0, stabilizing in later slices.
```

Comment:
some of these expected patterns like this and the below one, I'm not sure if they are really related to the input command above, or should just be bulleted use cases?